students/slawinski_amadeusz/lesson_06_dicts_tuples_sets_args_kwargs/08_uppercase.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+
+def capitalize(string):
+    capstring = ""
+    if string is not None:
+        firstchar = True
+        for char in string:
```

Comment:
jako że chcemy zmienić pierwszy znak każdego słowa w zdaniu, a każde słowo jest to `string`, który podajesz na wejściu tej funkcji to nie ma potrzeby iterowania po wszystkich elementach tego stringa. Wystarczy podmienić `string[0]`
można sobie ułatwić i wykorzystac funkcję `upper()` i cięcie i dodawanie stringów.